volkr_munko/src/Css/ContactStyled.js

```diff
@@ -0,0 +1,198 @@
+import styled from 'styled-components';
+
+export const ContactSection = styled.div`
+  padding: 140px 0;
+  background: linear-gradient(180deg, #f8f8f8, #fff);
+
+  @media (max-width: 991px) {
+    padding: 50px 20px;
+  }
+`;
+
+export const Container = styled.div`
+  width: 100%;
+  max-width: 1088px;
+  margin: 0 auto;
+`;
+
+export const ContactSectionInnerPart = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  @media (max-width: 991px) {
+    flex-direction: column;
+  }
+`;
+
+export const LeftInnerPart = styled.div`
+  max-width: 500px;
+  width: 100%;
+  position: relative;
+
+  @media (max-width: 991px) {
+    max-width: 550px;
+    width: 100%;
+  }
+`;
+
+export const LeftInnerMap = styled.div`
+  overflow: hidden;
+
+  iframe {
+    border-radius: 22px;
+    border-color: #f8f9fa;
+  }
+
+  img {
+    max-width: 40px;
+    width: 100%;
+    height: 40px;
+    position: absolute;
+    top: 214px;
+    right: 238px;
+  }
+
+  @media (max-width: 991px) {
```

Comment:
need this in img  { } because you have to write css for that only @aeshapatel025